ocs_ci/utility/service.py

```diff
@@ -34,6 +35,10 @@ def __init__(self, service_name, force=False):
 
         self.nodes = node.get_node_ip_addresses("InternalIP")
 
+        self.bastion_ip = "127.0.0.1"
+        if "bastion_ip" in config.ENV_DATA:
+            self.bastion_ip = config.ENV_DATA["bastion_ip"]
```

Comment:
```suggestion
        self.bastion_ip = config.ENV_DATA.get("bastion_ip", "127.0.0.1")
```

Can be just one liner